qemu/tests/driver_load.py

```diff
@@ -79,11 +79,24 @@ def get_driver_id(cmd, pattern):
         session.close()
         return driver_id
 
+    def service_operate(cmd):
+        """
+        Stop/Start service
+        :param cmd: cmd to stop/start service
+        """
+        session = vm.wait_for_login()
+        status, output = session.cmd_status_output(cmd)
+        session.close()
+        if status != 0:
+            raise exceptions.TestError("Service operation failed: %s" % output)
```

Comment:
Also as suggested by Avocado framework team to use test.error(), let us adopt it.
https://github.com/avocado-framework/avocado-vt/pull/966/files#diff-9938c1f55fadcbc7cf10cf03d5e6598eR165

Reply:
I think two options, 1. keep the same as old one, as there are other exceptions.TestError in this script
                               2. update all to test.error()